bridgemetadata/convert.py

```diff
@@ -98,4 +95,20 @@ def _hasTag(tag):
         elif "19115" in schemaName:
             return Format.ISO19139
     else:
-        return Format.FGDC
+        print ("Unknown source format")
+        return None
+
+def main():
+    if len(sys.argv) != 4:
+        print(
+            "Wrong number of parameters\nUsage: convert src dst dest-format"
+        )
+    else:
+        dst = None
+        for f in Format:
+            if str(f) == "Format."+sys.argv[3].upper():
+                dst = f
+        if (dst is None):
+            raise UnknownFormatException(str(dst))
+        convert(sys.argv[1], dst, None, sys.argv[2])
+        print("Done")
```

Comment:
Should we perhaps consider using the `argparse` lib for this? 
It automatically enforces the right number of parameters and can check the types as well.
Furthermore, we could place the contents of this function in a separate `__main__.py` script (then we can also remove the `if name == '__main__'` logic), so that `bridgemetadata` can be called as a Python module.